PhysicsTools/PatAlgos/python/slimming/MicroEventContent_cff.py

```diff
@@ -76,26 +76,37 @@
         'keep CTPPSLocalTrackLites_ctppsLocalTrackLiteProducer_*_*'
     )
 )
-MicroEventContentMC = cms.PSet(
-    outputCommands = cms.untracked.vstring(MicroEventContent.outputCommands)
-)
-MicroEventContentMC.outputCommands += [
-        #'keep *_slimmedGenJets*_*_*',
-        'keep *_slimmedGenJets_*_*',
-        'keep *_slimmedGenJetsAK8_*_*',
+
+MicroEventContentGEN = cms.PSet(
+    outputCommands = cms.untracked.vstring(
         'keep patPackedGenParticles_packedGenParticles_*_*',
         'keep recoGenParticles_prunedGenParticles_*_*',
         'keep LHEEventProduct_*_*_*',
-        'keep PileupSummaryInfos_slimmedAddPileupInfo_*_*',
         'keep GenFilterInfo_*_*_*',
         'keep GenLumiInfoHeader_generator_*_*',
         'keep GenLumiInfoProduct_*_*_*',
         'keep GenEventInfoProduct_generator_*_*',
+        'keep *_slimmedGenJets_*_*',
+        'keep *_slimmedGenJetsAK8_*_*',
+        'keep *_genMetTrue_*_*',
+        'keep *_genMetCalo_*_*',
+        'keep *_genMetCaloAndNonPrompt_*_*',
```

Comment:
I don't know if anybody is using genMetCalo, but anyway it's computed only from particles with |&eta;| < 5, so it should be possible to recompute it from the packed gen particles.